README.md

```diff
@@ -16,36 +19,37 @@ _[Demo and API Docs](https://elements.polymer-project.org/elements/iron-demo-hel
 
 ##&lt;demo-snippet&gt;
 
-
 `demo-snippet` is a helper element that displays the source of a code snippet and
 its rendered demo. It can be used for both native elements and
 Polymer elements.
 
-    Example of a native element demo
+```html
```

Comment:
Technically it's legal HTML! Plain text is totally fine in HTML.

If we don't want the text as part of these code samples we just need to deindent them in demo-snippet.html